src/is-my-room/test.ts

```diff
@@ -0,0 +1,31 @@
+import test from 'ava';
+
+import { isMyRoom } from './index';
+
+declare const global: any;
+
+function stubGame(){
+	global.Game = {
+		rooms: {
+			'W1N1': {
+				controller: {
+					my: true
+				}
+			},
+			'W2N1': {
+				controller: {
+					my: false
+				}
+			},
+			'W3N1': {}
+		}
+	}
+}
+
+test('Should only return true for owned rooms', t => {
+	stubGame();
+
+	t.true(isMyRoom('W1N1'));
+	t.false(isMyRoom('W2N1'));
+	t.false(isMyRoom('W3N1'));
```

Comment:
You might want to add a test whether this works for rooms without vision.

Reply:
Good catch, added some tests for that.